RELEASING.md

```diff
@@ -63,3 +63,21 @@
   * https://github.com/scalameta/metadoc
   * https://github.com/scalameta/metals
   * https://github.com/sbt/sbt (see `SemanticdbPlugin`)
+
+## Releasing for a specific version
+
+Realeasing for a specific existing version, which is needed for example in case of new Scala version, is more complicated. The flow is as follows:
```

Comment:
Have you tried to release it using a special tag?
I added some changes to workflow but never tested them.

Reply:
I forgot about them :thinking: but I managed to run it locally for now. We can test it out for the new release.

Reply:
ok!
Honestly, I'm not 100% sure that everything will be fine :smile: 
I will check it on previous scalameta versions a bit later